big/int.go

```diff
@@ -0,0 +1,270 @@
+package big
```

Comment:
From https://github.com/filecoin-project/go-state-types with a few cleanups. Importantly, nil is no longer a half-way alternative to zero...